programs/governance-rewards/src/state/distribution.rs

```diff
@@ -0,0 +1,70 @@
+use anchor_lang::prelude::*;
+
+use super::distribution_option::{DistributionOption, DistributionOptions};
+
+#[account]
+#[derive(Debug, PartialEq, Eq)]
+pub struct Distribution {
+    pub registration_period_end_ts: u64,
+    pub voter_weight_program: Pubkey,
+    pub realm: Pubkey,
+    pub registrar: Option<Pubkey>,
```

Comment:
this field looks unused

Reply:
It's used by the crank to find eligible voters.

Reply:
ah, makes sense! maybe add a comment about that

Reply:
Done